test/compilable/ddoc17697.d

```diff
@@ -0,0 +1,15 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o-
+// POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh 17697
+
+/***
+ * See:
+ *    http://www.fooa.com/test1
```

Comment:
I think Adam's request is valid. There is no test that ensures that any underscores prefixing the documented symbol's identifier will be preserved in the URL.